start.php

```diff
@@ -83,11 +83,25 @@ function_exists( 'wp_is_json_request' ) &&
      */
 	$themes_directory         = get_theme_root( get_stylesheet() );
 	$themes_directory_name    = basename( $themes_directory );
-	$theme_candidate_basename = basename( dirname( $fs_root_path ) ) . '/' . basename( $fs_root_path );
 
-	if ( $file_path == fs_normalize_path( realpath( trailingslashit( $themes_directory ) . $theme_candidate_basename . '/' . basename( $file_path ) ) )
-	) {
-		$this_sdk_relative_path = '../' . $themes_directory_name . '/' . $theme_candidate_basename;
+	// This change ensures that the condition works even if the SDK is located in a subdirectory (e.g., vendor)
+	$theme_candidate_basename = str_replace($themes_directory . '/' . get_stylesheet() . '/', '', $fs_root_path );
+
+	// Check if the current file is part of the active theme.
+	$is_current_active_theme = $file_path == $themes_directory . '/' . get_stylesheet() . '/' . $theme_candidate_basename . '/' . basename($file_path);
```

Comment:
I would name the variable like

1. `$is_current_active_theme` --> `$is_current_sdk_from_active_theme`
2. `$is_current_parent_theme` --> `$is_current_sdk_from_parent_theme`.